Classes/common/touchdb/ChangeTracker/TDURLConnectionChangeTracker.m

```diff
@@ -81,17 +86,18 @@ - (BOOL)start
         }
     }
     
-    self.connection = [[NSURLConnection alloc] initWithRequest:self.request
-                                                      delegate:self
-                                              startImmediately:NO];
+    self.task = [self.session dataTaskWithRequest:self.request completionHandler:^(NSData *data, NSURLResponse *response, NSError *error) {
+        if(error){
+            [self failedWithError:error];
+            return;
+        } else {
+            [self receivedResponse:response];
+            [self receivedData:data];
+            [self finishedLoading];
+        }
+    }];
```

Comment:
I'm not sure, I've converted it to use the normal `self` to `weakSelf` to `strongSelf` pattern in d1e9e05